api/slash-z-inner.js

```diff
@@ -0,0 +1,242 @@
+import Prisma from './prisma.js'
+import isPublicSlackChannel from './is-public-slack-channel.js'
+import userIsRestricted from './user-is-restricted.js'
+import channelIsForbidden from './channel-is-forbidden.js'
+import openZoomMeeting from './open-zoom-meeting.js'
+import transcript from './transcript.js'
+import fetch from 'node-fetch'
+
+export default async ({
```

Comment:
If this file isn't taking a `res` & `req`, maybe we can bring it up directly into `api/`?

Reply:
Wdym?

Reply:
nvm on this! new gh filetree was messing with me– thought it was inside `api/endpoints/slash-z-inner.js`

Reply:
haha, happens!